ow_lander/scripts/activity_full_digging_traj.py

```diff
@@ -132,12 +132,14 @@ def dig_linear(move_arm,move_limbs,args):
   #rotate dist pith to pre-trenching position.
   change_joint_value(move_arm, constants.J_DIST_PITCH, -math.pi/2)
 
-  ## Once aligned to trench goal, place hand above trench middle point
-  z_start = ground_position + constants.SCOOP_OFFSET - depth
-  go_to_Z_coordinate(move_limbs, x_start, y_start, z_start)
+  ## Once aligned to trench goal, place hand above the desired start point
+  alpha = math.atan2(constants.WRIST_SCOOP_PARAL,constants.WRIST_SCOOP_PERP)
```

Comment:
Try to be consistent with your whitespace. I find things to be more readable if there are spaces after commas and spaces around mathematical operators. You do it both ways here.